topics/materials-science/tutorials/muspinsim/tutorial.md

```diff
@@ -0,0 +1,932 @@
+> 2. Open the editor for each workflow, and verify that all the required tools
+>    are installed on your Galaxy instance, and have the same version(s) as the
+>    tools in workflow. If this is not the case a dialogue box will warn you.
+>
+{: .hands_on}
+
+# Workflow 1 - Zeeman Coupling
+
+The Zeeman interaction is the interaction of any spin with an external magnetic
+field. In MuSpinSim this mainly means a strong global field applied to the
+entire experiment, though it is possible for the user to also set custom local
+magnetic fields for individual spins (representing for example the effect of
+nearby paramagnetic nuclei). The Hamiltonian written explicitly is:
+
+$$
+\mathcal{H}_Z = 2\pi\hbar\sum_i^N \gamma_i (\mathbf{B}+\mathbf{B}_i)\mathbf{S}_i
+$$
+
+The sum is intended over all spins, with the gyromagnetic ratios depending on
+the nuclei's own properties. The magnetic field here was split in two parts, a
+global and a local one which bears the index of the spin. Of course, both of
+these can be zero, and by default are if the user does not specify anything.
+$$\mathbf{S}_i$$ is the vector of spin operators.
+
+We can execute the workflow that uses Zeeman coupling in one go, but to aid
+understanding we will describe each step in detail below.
+
+## MuSpinSim Configure
+
+The first step is to generate a configuration file for the rest of the process.
+
+> <hands-on-title>Tool details</hands-on-title>
+>
+> 1. {% tool [MuSpinSim Configure](toolshed.g2.bx.psu.edu/repos/muon-spectroscopy-computational-project/muspinsim_config/muspinsim_config/1.1.0+galaxy0) %} with the following parameters:
+>    - *"Name"*: `zeeman`
+>    - *"Spins"*:
+>      - *"Spins to simulate"*:
+>        - *"Spins to simulate 1"*:
+>          - *"Species"*: `mu`
+>    - *"Spin Interactions"*:
+>      - *"Interactions to simulate"*:
+>        - *"Interactions to simulate 1"*:
+>          - *"Choose interaction type"*: `Zeeman`
+>          - *"Index of coupled spin"*: `1`
+>          - *"Zeeman coupling vector"*: `0 0 20.0/muon_gyr`
+>    - *"Experiment Parameters"*:
+>      - *"Experiment type"*: `custom`
+>      - *"X axis"*: `time`
+>      - *"Y axis"*: `asymmetry`
+>      - *"Average axes"*: `orientation`
+>      - *"Euler Convention"*: `ZYZ`
+>      - *"Times"*:
+>        - *"Time 1"*:
+>          - *"Time"*: `range(0, 0.1, 100)`
+>    - *"Fitting Parameters"*:
+>      - *"Fit experimental data with simulations"*: `No`
+> 2. Should generate the following output(s):
+>    - {% icon param-file %} `muspinsim input file zeeman` (Output dataset)
+>
+{: .hands_on}
+
+The output of this tool is a `txt` file, formatted for MuSpinSim to use for the
+the simulation itself. While there are a lot of parameters being defined, we can
+focus on the *"Interactions to simulate"* section. Here we define only one
+coupling, Zeeman, and we define its direction (aligned with the $$z$$ axis),
+strength ($$20.0$$) and units. The coupling value is implicitly assumed to be in
```

Comment:
This is another one that's inherited from where I copied the content from: [examples.md](https://github.com/muon-spectroscopy-computational-project/muspinsim/blob/main/docs/docs/examples.md) I don't have any strong feelings on this, other than that the approach should be consistent (i.e. if we use them in the MSS docs we should use them here, if we don't want to use them here then we shouldn't in the MSS docs).

Reply:
+1 for consistency. It looks like the MSS docs only use TeX where symbols are needed (most obvious in example 5).

Reply:
So approach should be:
 - Use TeX for equations involving symbols: `$\gamma_\mu B = 20 \,\mathrm{MHz}$`
 - Use TeX for variables represented by symbols: `$\Delta_1$`
 - Don't use TeX for values only using roman: `135.54 MHz`
 - Don't use TeX for variables only using roman: `low T`, `high T`
 - Use `\mathrm{}` on units that appear in TeX

To me there are a couple of less clear cases:
 - Values that use greek letters: `$0.1\,\mu s$`
   - [ ] I suppose we have to use TeX as we need the `\mu` (though in this case we should use `\mathrm{}` as well
   - [ ] Alternatively could use `u` as a stand in for `$\mu$`
   - [ ] Could try only TeXing the `\mu`, i.e. `0.1$\mathrm{\mu}$s`, but not sure if that looks good or not
 - Equations that only use roman: `$T = 1 \,\mathrm{K}$`
   - [ ] Currently this uses TeX like other equations meaning
   - [ ] Alternatively could just just write it plain (but it would then look weird next to `$T=\infty$` which needs the infinity symbol

Preferences?

Reply:
µ exists in Unicode so it should work fine in plaintext. So `0.1 µs`.

For the second one, stick with TeX. I think there's TeX wherever `=` is used (it is a symbol, after all).